db/schema.rb

```diff
@@ -53,6 +55,13 @@
     t.index ["team_id"], name: "index_pager_duty_calendars_on_team_id"
   end
 
+  create_table "spatial_ref_sys", primary_key: "srid", id: :integer, default: nil, force: :cascade do |t|
```

Comment:
I know this is from a dump of the prod DB, but what's this table for?

Reply:
From the default extensions that are installed in databases created by the RDS broker